assemblies/assembly-customizing-the-global-header.adoc

```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: customizing-the-global-header
+[id="assembly-customizing-red-hat-developer-hub-global-header"]
+= Customizing the {product} global header
+
+As an administrator, you can enable or disable the {product} global header to create a consistent and flexible navigation bar across your {product-custom-resource-type} instance. You can extend the global header with additional buttons and customize the order and position of icons and features. By default, the {product} global header includes a search field, create, support and notification buttons, and a user profile drop-down.
```

Comment:
Consider refining the scope for consistency: 

* The introduction promise is to *extend* the global header. Logically, you would expect to continue using the default global header, and learn how to modify its behavior.
* The procedure explains how to *replace* the global header entirely, with an custom plugin (but does not explain how to create the plugin). 

Additionnally, the corresponding engineering task is not completed: https://issues.redhat.com/browse/RHIDP-4659. Maybe we should wait for the engineering feature before documenting?